content/posts/2024/2024-11-11-openQA_and_PowerPC.md

```diff
@@ -0,0 +1,69 @@
+---
+title: openQA and PowerPC
+author: phoenix
+type: post
+date: 2024-11-11T11:14:48+01:00
+categories:
+  - openqa
+tags:
+  - openqa
+  - PowerPC
+---
+Due to recent changes in the worker configuration, we needed to reconfigure some of the PowerPC jobs in openQA. This triggered a couple of questions regarding the availability of openQA worker, worker backends, their differences and their caveats. This blog post should act as a quickstart/overview guide for people getting into OpenQA testing on the PowerPC architecture.
```

Comment:
You might want to clarify that actually no changes were done in upstream openQA. You are talking about a SUSE-internal infrastructure change only.